### DISCUSSION TITLE
Import of old Settings and Combination Files

### DISCUSSION THREAD

**Discussion by bikerparkes** [GrandOrgue/grandorgue#1631] [Help]
I have just upgraded from Grandoegue v0.3.1.2330 to the current v3.12.2-1 with the hope of using some of the newer sample sets correctly and in particular Lars Paolo's updated Burea Extended set. The new Grande Orgue appears to be working correctly but doesn't appear to load my settings file, so it looks like I would have to reset the recognition of my keyboards etc and reprogram all my Divisional , general and stepper settings. In the drop down menu I see that Settings and Combinations are now exported separately, but importing my old file simply produces a set of errors of the form "Warning unused CMB entry Setter......" and doesn't do anything.

Is there any way to transfer from the old *.CMB to the new types, or am I going to have to reprogram it all again?

This may have been covered when the the software changed, but I can't find anything looking like it in the discussions, and realise it may have been some time ago, but I've had no reason to update a functioning system.

Thanks

**Comment by jason-barba** [comment]
See this post https://github.com/GrandOrgue/grandorgue/discussions/1399

//Jens
